docs/mobile-apps/live-testing/testing-apple-pay.md

```diff
@@ -0,0 +1,158 @@
+
+
+## Testing Apple Pay
+
+Sauce Labs offers three ways to test Apple Pay :
+
+- Using Simulators.
+- Using real private devices with an Apple Pay Sandbox Testing account.
+- Using real private devices with a real production account and real credit cards.
+
+:::caution iOS Simulators
+There are important differences between the Apple Pay Real Device and Simulator flow. The Simulator has the following limitations:
+
+- It is focused on the front-end integration of Apple Pay and does not test the back-end integration.
+- You **can't** add cards to the wallet, meaning:
+  - No Apple Pay Sandbox Testing cards.
+  - No real credit cards.
+- You **can't** test the Apple Pay in-web flow.
+- You **can** test the Apple Pay in-app flow, but the Apple Pay in-app flow will not work the same as with Real Devices. It won't return a payment token and will not properly process your payment. In addition to this, it automatically provides simulated cards for all the supported payment networks.
+
+:::
+
+## Requirements
+
+- You need to use [Private devices](#apple-pay-on-real-private-devices).
+- [Instrumentation](#disable-instrumentation) needs to be disabled.
+- You need to add your Sauce Labs hosted Private device [UDID](#apple-pay-on-real-private-devices) to your own provisioning profile.
+- Devices need to have a physical home button (for instance, iPhone SE(2020/2022)/6 series/7 series/8 series). A physical button will require the passcode for payment confirmation.
+
+:::note
+Devices with a notch (like the iPhone X(S)/11/12/..) will ask for FaceId confirmation. As this feature is disabled, the payment will fall back to a different confirmation method. It will use a double press on the power button for the payment approval. However, this method is not supported.
+:::
+
+## Apple Certificates
+
+Apple certificates are used to ensure security in their systems, and they are much more strict about them than Android. This level of security makes certificates a very complex part of making Apple Pay work with devices in a cloud.
+
+To give you an example, Android apps can be installed without any specific signing on whatever real device you want. With Apple you have two options, or you need to add a remote device to your developer certificate and the provisioning profile, so you are allowed to install the app on that specific device. Or you need to use an enterprise certificate where the Apple device that has that certificate installed allows you to install the app. Similarly, when you install an iOS app on a device, we re-sign the app with a Sauce Labs enterprise certificate so you can install your app on all Sauce Labs public/private devices.
+
+:::note
+Apple Pay has a limitation that it cannot work with an enterprise certificate. You need to use the developer certificate where the device has been added to the provisioning profile in order to make this work. This can only be done for Sauce Labs private devices on which you have disabled the instrumentation.
```

Comment:
```suggestion
Apple Pay has a limitation that it cannot work with an enterprise certificate. You need to use the developer certificate where the device has been added to the provisioning profile to make this work. This can only be done for Sauce Labs private devices on which you have disabled the instrumentation.
```